lib_lua/CMLinearL2AutoLambda.lua

```diff
@@ -0,0 +1,162 @@
+--[[ Description: CLinear class implements linear GNN module with multiplicative terms.
+]]
```

Comment:
You should be reusing functions from CMLinearL2 here instead of copying a lot of code.